docs/configuration/retries.md

```diff
@@ -1,117 +1,3 @@
-            }
-            catch (Exception e) when (++tries < maxTries && filter(e))
-            {
-            }
-        }
-    }
-
-    private async Task<T> Try<T>(Func<Task<T>> operation, CancellationToken token)
-    {
-        for (var tries = 0; ; token.ThrowIfCancellationRequested())
-        {
-            try
-            {
-                return await operation();
-            }
-            catch (Exception e) when (++tries < maxTries && filter(e))
-            {
-            }
-        }
-    }
-}
-```
-<sup><a href='https://github.com/JasperFx/marten/blob/master/src/Marten.Testing/Examples/RetryPolicyTests.cs#L12-L90' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_retrypolicy-samplepolicy' title='Start of snippet'>anchor</a></sup>
-<!-- endSnippet -->
-
-The policy is then plugged into the `StoreOptions` via the `RetryPolicy` method:
-
-<!-- snippet: sample_retrypolicy-samplepolicy-pluggingin -->
-<a id='snippet-sample_retrypolicy-samplepolicy-pluggingin'></a>
-```cs
-// Plug in our custom retry policy via StoreOptions
-// We retry operations twice if they yield and NpgsqlException that is transient
-c.RetryPolicy(ExceptionFilteringRetryPolicy.Twice(e => e is NpgsqlException ne && ne.IsTransient));
-```
-<sup><a href='https://github.com/JasperFx/marten/blob/master/src/Marten.Testing/Examples/RetryPolicyTests.cs#L100-L104' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_retrypolicy-samplepolicy-pluggingin' title='Start of snippet'>anchor</a></sup>
-<!-- endSnippet -->
-
-Lastly, the filter is configured to retry failing operations twice, given they throw a `NpgsqlException` that is transient and thus might succeed later.
-
-There's also a built-in `DefaultRetryPolicy` that has sane defaults for transient error handling. Like any custom policy, you can plug it into into the `StoreOptions` via the `RetryPolicy` method:
-
-<!-- snippet: sample_retrypolicy-samplepolicy-default -->
-<a id='snippet-sample_retrypolicy-samplepolicy-default'></a>
-```cs
-// Use DefaultRetryPolicy which handles Postgres's transient errors by default with sane defaults
-// We retry operations twice if they yield and NpgsqlException that is transient
-// Each error will cause sleep of N seconds where N is the current retry number
-c.RetryPolicy(DefaultRetryPolicy.Twice());
-```
-<sup><a href='https://github.com/JasperFx/marten/blob/master/src/Marten.Testing/Examples/RetryPolicyTests.cs#L106-L111' title='Snippet source file'>snippet source</a> | <a href='#snippet-sample_retrypolicy-samplepolicy-default' title='Start of snippet'>anchor</a></sup>
-<!-- endSnippet -->
-
-Also you could use the fantastic [Polly](https://www.nuget.org/packages/polly) library to easily build more resilient and expressive retry policies by implementing `IRetryPolicy`.
+Marten's previous, homegrown `IRetryPolicy` mechanism was completely replaced by  [Polly](https://www.nuget.org/packages/polly) in Marten V7.
```

Comment:
**SUGGESTION:** It'd be good to provide some samples on how to plug Polly policies to Marten.

Reply:
Yeah, I agree. Like I told y'all in Discord, I hadn't made any effort to document anything yet.